ci/test-mill-1.sh

```diff
@@ -6,4 +6,6 @@ set -eux
 git clean -xdf
 
 # Run tests
-mill integration.test "mill.integration.local.{JawnTests,BetterFilesTests,UpickleTests}"
+./mill integration.test "mill.integration.local.JawnTests"
+./mill integration.test "mill.integration.local.BetterFilesTests"
+./mill integration.test "mill.integration.local.UpickleTests"
```

Comment:
Not sure why this stuff is necessary, but it seems to make tests pass. Can investigate in a separate PR